usermods/TetrisAI_v2/usermod_v2_tetrisai.h

```diff
@@ -96,6 +94,8 @@ void drawGrid(TetrisAIGame* tetris, TetrisAI_data* tetrisai_data)
 ////////////////////////////
 uint16_t mode_2DTetrisAI()
 {
+  static unsigned long lastTime = 0;
```

Comment:
Don't use static variables in an effect function. 
You can either use `SEGENV.aux0`, `SEGENV.aux1`, `SEGENV.step`, or allocate persistent data by using `SEGENV.allocateData(dataSize)`.

Reply:
You can find many good examples for this in wled00/FX.cpp

Reply:
I started to create a 'non blocking' version of the AI part. The commits I took back were related to that.

I need to have a look in the 'static' part.